nccr-mediality.csl

```diff
@@ -310,8 +300,9 @@
         </else-if>
         <else-if position="subsequent">
           <text macro="contributors-short" suffix=", "/>
-          <text macro="title-short"/>
-          <text variable="first-reference-note-number" prefix=" (Anm. " suffix=")"/>
+          <text variable="title" form="short"/>
+          <text term="note" form="short" prefix=" (" suffix="."/>
+          <text variable="first-reference-note-number" prefix=" " suffix=")"/>
```

Comment:
@cnotz, it's better to replace

```xml
          <text term="note" form="short" prefix=" (" suffix="."/>
          <text variable="first-reference-note-number" prefix=" " suffix=")"/>
```

by

```xml
        <group prefix=" (" suffix=")" delimiter=" ">
          <text term="note" form="short"/>
          <text variable="first-reference-note-number"/>
        </group>
```

and change

```
<term name="note" form="short">Anm</term>
```

by

```
<term name="note" form="short">Anm.</term>
```

That clarifies how the parentheses belong together (it's a good habit to use groups wherever possible in styles for placing punctuation). (this `<group/>` would also prevent the "note" term from being printed if the "first-reference-note-number" variable doesn't have a value, although I guess that shouldn't really be possible for `position="subsequent"`)

In our world of CSL we also always define abbreviated terms with periods.